_includes/topnav.html

```diff
@@ -2,84 +2,82 @@
-	                    })
-	                    </script>
-	                    <!--end search-->
-	                </li>
-	            </ul>
-	        </div>
+	<nav class="TopNavItem">
+        <div class="NavGroup">
+            <ul class="NavList horizontal">
+                <!-- entries without drop-downs appear here -->
+                {% for entry in site.data.topnav.topnav %}
+                {% for item in entry.items %}
+                {% if item.external_url %}
+                <li class="NavLinkLi">
+                	<a href="{{item.external_url}}" target="_blank" class="NavLink">{{item.title}}</a>
+                </li>
+                {% elsif page.url contains item.url %}
+                <li class="active NavLinkLi isActive">
+                	<a href="{{item.url | remove: "/"}}" class="NavLink">{{item.title}}</a>
+                </li>
+                {% else %}
+                <li class="NavLinkLi">
+                	<a href="{{item.url | remove: "/"}}" class="NavLink">{{item.title}}</a>
+               	</li>
+                {% endif %}
+                {% endfor %}
+                {% endfor %}
+                <!-- entries with drop-downs appear here -->
+                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
+                {% for entry in site.data.topnav.topnav_dropdowns %}
+                {% for folder in entry.folders %}
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ folder.title }}<b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        {% for folderitem in folder.folderitems %}
+                        {% if folderitem.external_url %}
+                        <li><a href="{{folderitem.external_url}}" target="_blank">{{folderitem.title}}</a></li>
+                        {% elsif page.url contains folderitem.url %}
+                        <li class="dropdownActive"><a href="{{folderitem.url |  remove: "/"}}">{{folderitem.title}}</a></li>
+                        {% else %}
+                        <li><a href="{{folderitem.url | remove: "/"}}">{{folderitem.title}}</a></li>
+                        {% endif %}
+                        {% endfor %}
+                    </ul>
+                </li>
+                {% endfor %}
+                {% endfor %}
+                {% if site.feedback_disable == null or site.feedback_disable == false %}
+			{% include feedback.html %}
+		{% endif %}
+
+                <li>
+                    <!--start search-->
+                    <div id="search-demo-container">
+                        <input type="text" id="search-input">
+                        <i class="fa fa-search"></i>
+                        <ul id="results-container"></ul>
+                    </div>
+                    <script src="{{ "js/jekyll-search.js"}}" type="text/javascript"></script>
```

Comment:
Not required for this PR, but we should start collecting all the JS from the templates into a single JS file.

Reply:
I'll include this search JS in the search task.